### ISSUE TITLE
Cannot find file transferred using SSH SFTP sampler

### ISSUE THREAD

**Issue by jmelom-spok** [yciabaud/jmeter-ssh-sampler#11]
Running JMeter from a Windows 10 system, I can see the contents of the transferred file in the Results tab in the View Results Tree listener, but I cannot find the contents of the transferred file actually saved to disk.  I tried specifying a destination path for the file and not specifyiing a destination path hoping that would let the file be saved in a default location.  I could not find a saved file in either case.

**Comment by jmelom-spok** [comment]
I believe I have figured out my problem.  
When "Print file content" is True, then a file is not saved.
When "Print file content" is False, then the "Destination path" is used to specify where to save a file.

I had been running with Print file content = True and Destination path set to a file path.

I set Print file content = False.  Then the file was copied to the location I specified in the Destination path.

A more careful reading of the instructions led me to try this.  After my first reading it was not clear to me that "Print file content" governed an "exclusive or"  for saving the file.
